test/clj/game/test_framework.clj

```diff
@@ -44,11 +44,26 @@
 (load-all-cards)
 
 (defn is-hand?
+  "Is the hand exactly equal to a given set of cards?"
   [state side expected-hand]
   (let [expected-hand (seq (sort (flatten expected-hand)))
         hand (seq (sort (map :title (get-in @state [side :hand]))))]
     (is (= expected-hand hand) (str "hand is not " expected-hand))))
 
+(defn is-deck?
+  "Is the discard exactly equal to a given set of cards?"
+  [state side expected-deck]
+  (let [expected-deck (seq (sort (flatten expected-deck)))
+        deck (seq (sort (map :title (get-in @state [side :deck]))))]
+    (is (= expected-deck deck) (str "deck is not " expected-deck))))
+
+(defn is-discard?
+  "Is the set of cards in the deck exactly equal to a given set of cards? (this is order agnostic)"
+  [state side expected-discard]
+  (let [expected-discard (seq (sort (flatten expected-discard)))
+        discard (seq (sort (map :title (get-in @state [side :discard]))))]
```

Comment:
Should deck and discard be sorted? Hand was sorted because it's an unordered zone, but deck and discard are ordered.

Reply:
The intention for these is just to check the set of cards is the same. I could change the name if it's unclear.

Reply:
It's probably fine and I'm just needlessly nitpicky lol